tests/UI/campaigns/utils/setup.js

```diff
@@ -37,3 +39,21 @@ after(async function () {
     await files.createFile('.', `${reportName}.json`, JSON.stringify(browserErrors));
   }
 });
+
+
+afterEach(async function () {
+  // Take screenshot if demanded after failed step
+  if (global.TAKE_SCREENSHOT_AFTER_FAIL && this.currentTest.state === 'failed') {
+    const currentTab = await helper.getLastOpenedTab(this.browser);
+
+    // Take a screenshot
+    await currentTab.screenshot(
+      {
+        path: `./screenshots/fail_test_${screenshotNumber}.png`,
```

Comment:
Could we put the identifier somehow ?

Reply:
I think using the step identifier as file name can be very bad if it's too long.

Let me explain why I used this formula:
When running mocha in terminal, The first fail step marks `1)`, the second fail step `2)`... like in this capture
![image](https://user-images.githubusercontent.com/48441421/103632458-9caebd80-4f44-11eb-9908-e9a8c2a388ed.png)


So, Using the same number, screenshots will be ordered and the user will easily know that first screenshot is for the first fail step ...